### DISCUSSION TITLE
Centos 8 zfs repo version is 0.8.6 instead of 2.0.4

### DISCUSSION THREAD

**Discussion by mkiah** [openzfs/zfs#12202] [Q&A] [answered]
Adding the zfsonlinux repo to Centos8, the newest available version is 0.8.6, while the newest release on this repo is 2.0.4.

To see the 2.0.4 version, I need to enable the zfs-testing repo. This seems counter-intuitive if 2.0.4 is the new stable release, and the documentation says [not to use zfs-testing in production](https://openzfs.github.io/openzfs-docs/Getting%20Started/RHEL%20and%20CentOS.html#testing-repositories).

I'm probably just missing something, does someone mind filling me in? 

Thanks in advance!

**Comment by rincebrain** [comment, marked as answer]
[This](https://github.com/openzfs/zfs/issues/12234#issuecomment-867226856) answers your question in full, I think.
